src/memorial_logger.cpp

```diff
@@ -672,6 +672,14 @@ void memorial_logger::notify( const cata::event &e )
             }
             break;
         }
+        case event_type::cuts_tree: {
+            character_id ch = e.get<character_id>( "character" );
+            if( ch == g->u.getID() ) {
+                add( pgettext( "memorial_male", "Fell a tree." ),
+                     pgettext( "memorial_female", "Fell a tree." ) );
+            }
+            break;
+        }
```

Comment:
This should be in the past tense.  So `Felled a tree`.  Or just exclude it entirely.  Not sure if felling a tree is significant enough to warrant a place in the memorial log.  I don't have a string opinion either way.

Reply:
I think it should be excluded.